### PR TITLE
feat: Engagement Center Change application's name to contributions - MEED-604 - Meeds-io/meeds#13

### PR DESCRIPTION
**Pull request by ArfaouiMohamedAmine**
In order to introduce our new application `Contributions` in app center, many modifications has been applied, 
prior to this change is was named `Challenges`, this change is going to change the application name to `Contributions` .

### CODE REVIEW

File: portlets/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue [hunk 1]
```diff
@@ -96,6 +96,8 @@ export default {
       this.tab = 1;
     } else if  (urlPath.indexOf(`${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions/achievements`) > -1) {
       this.tab = 2;
+    } else if (!this.engagementCenterEnabled) {
+      window.history.pushState('Engagement Center', this.$t('engagementCenter.label.contributions'), `${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions`);
     } else {
       this.tab = 0;
       window.history.pushState('Engagement Center', this.$t('engagementCenter.label.programs'), `${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions/programs`);
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue]:
```suggestion
```

File: portlets/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml [hunk 2]
```diff
@@ -56,13 +56,13 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                     <description>description</description>
                     <object type="org.exoplatform.appcenter.dto.Application">
                         <field name="title">
-                            <string>Challenges</string>
+                            <string>Contributions</string>
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml:59]: Add translated label please! https://github.com/Meeds-io/app-center/blob/develop/app-center-services/src/main/resources/locale/addon/appcenter_en.properties

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml (hunk 2 above)
Comment (ArfaouiMohamedAmine) [reply portlets/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml:59]: the required modification has been treated in this pr https://github.com/Meeds-io/app-center/pull/244
thank you .

File: portlets/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue (hunk 1 above)
Comment (ArfaouiMohamedAmine) [reply portlets/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue]: done .

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**